snowflake-oauth: document where to locate the redirect URI (#5955)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md

dbt Cloud Enterprise supports [OAuth authentication](https://docs.snowflake.net/manuals/user-guide/oauth-intro.html) with Snowflake. When Snowflake OAuth is enabled, users can authorize their Development credentials using Single Sign On (SSO) via Snowflake rather than submitting a username and password to dbt Cloud. If Snowflake is setup with SSO through a third-party identity provider, developers can use this method to log into Snowflake and authorize the dbt Development credentials without any additional setup.

To set up Snowflake OAuth in dbt Cloud, admins from both are required for the following steps:
1. [Locate the redirect URI value](#locate-the-redirect-uri-value) in dbt Cloud.
2. [Create a security integration](#create-a-security-integration) in Snowflake.
3. [Configure a connection](#configure-a-connection-in-dbt-cloud) in dbt Cloud.

To use Snowflake in the dbt Cloud IDE, all developers must [authenticate with Snowflake](#authorize-developer-credentials) in their profile credentials.

### Locate the redirect URI value

> To get started, locate the connection's redirect URI in dbt Cloud:

Reviewer: ```suggestion
To get started, copy the connection's redirect URI from dbt Cloud:
```

> 1.  Navigate to **Account settings**

Reviewer: ```suggestion
1. Navigate to **Account settings**
```

dbt Cloud Enterprise supports [OAuth authentication](https://docs.snowflake.net/manuals/user-guide/oauth-intro.html) with Snowflake. When Snowflake OAuth is enabled, users can authorize their Development credentials using Single Sign On (SSO) via Snowflake rather than submitting a username and password to dbt Cloud. If Snowflake is setup with SSO through a third-party identity provider, developers can use this method to log into Snowflake and authorize the dbt Development credentials without any additional setup.

> To set up Snowflake OAuth in dbt Cloud, a Snowflake admin must:

Reviewer: ```suggestion
To set up Snowflake OAuth in dbt Cloud, admins from both are required for the following steps:
```

1. [Locate the redirect URI value](#locate-the-redirect-uri-value) in dbt Cloud.
2. [Create a security integration](#create-a-security-integration) in Snowflake.
3. [Configure a connection](#configure-a-connection-in-dbt-cloud) in dbt Cloud.

> To use Snowflake in the dbt Cloud IDE, all developers must [Authenticate with Snowflake](#authorize-developer-credentials) in their profile credentials.

Reviewer: ```suggestion
To use Snowflake in the dbt Cloud IDE, all developers must [authenticate with Snowflake](#authorize-developer-credentials) in their profile credentials.
```

### Locate the redirect URI value

> To get started, locate the connection's redirect URI for configuring Snowflake OAuth:

Reviewer: ```suggestion
To get started, locate the connection's redirect URI in dbt Cloud:
```

>   - Navigate to **Account settings**

Reviewer: ```suggestion
1.  Navigate to **Account settings**
```

>   - Select **Projects**

Reviewer: ```suggestion
1. Select **Projects** and choose a project from the list. 
```

| ENABLED  | Required |
| OAUTH_CLIENT  | Required |
| OAUTH_CLIENT_TYPE  | Required |

> | OAUTH_REDIRECT_URI  | Required. Use the value that you [located earlier in dbt Cloud](#locate-the-redirect-uri-value). |

Reviewer: ```suggestion
| OAUTH_REDIRECT_URI  | Required. Use the value in the [dbt Cloud account settings](#locate-the-redirect-uri-value). |
```
